apps/search-poc/src/components/Details.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { Spin } from 'antd';
+
+
+const DetailsComponent: React.FunctionComponent<{
+  data: any;
```

Comment:
`any` is bad! :wink: 

Also I'd prefer to see something more explicit like:
```suggestion
  id: string;
  name: string;
  description: string;
  brainRegion: {
    id: string;
    label: string;
  };
```

Reply:
I've also added types for common resources